org.eclipse.lemminx/pom.xml

```diff
@@ -375,5 +375,11 @@
 			<groupId>org.apache.velocity</groupId>
 			<artifactId>velocity-engine-core</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Shall we define the dependency in the dependencyManagement section of parent pom and use it here?

Reply:
Fixed in 709d06261071762bae9e719c65564ba23304ec47